modules/understanding-vulnerability-scores.adoc

```diff
@@ -6,26 +6,26 @@
 = Understanding vulnerability scores
 
 [role="_abstract"]
-In the RHACS portal shows a single Common Vulnerability Scoring System (CVSS) base score for each vulnerability.
+The {product-title-short} portal shows a single Common Vulnerability Scoring System (CVSS) base score for each vulnerability.
 {product-title} shows the CVSS score based on the following criteria:
 
 * If a CVSS v3 score is available, {product-title} shows the score and lists `v3` along with it.
 For example, `6.5 (v3)`.
 +
 [NOTE]
 ====
-CVSS v3 scores are only available if you are using Scanner version 1.3.5 and newer.
+CVSS v3 scores are only available if you are using the StackRox Scanner version 1.3.5 and newer or Scanner V4.
 ====
 * If a CVSS v3 score is not available, {product-title} shows only the CVSS v2 score.
 For example, `6.5`.
 
 You can use the API to get the CVSS scores.
-If CVSS v3 information is available for a Common Vulnerabilities and Exposures (CVE), the response includes both CVSS v3 and CVSS v2 information.
+If CVSS v3 information is available for a Common Vulnerabilities and Exposure (CVE), the response includes both CVSS v3 and CVSS v2 information.
 
-For some CVEs, the Red Hat Security Advisory (RHSA) CVSS score may differ from the CVSS score visible in the RHACS portal.
+For some CVEs, the Red Hat Security Advisory (RHSA) CVSS score may differ from the CVSS score visible in the {product-title-short} portal.
```

Comment:
This sentence doesn't really make sense. I'd say we should remove this sense and replace it with `For a Red Hat Security Advissory (RHSA), the CVSS score is set to the highest CVSS score among all the related CVEs`.